### PR TITLE
Project Motivations

### PR DESCRIPTION
**Pull request by theninj4**
This adds a section to the README covering the motivations behind this project.

### CODE REVIEW

**Comment by connormeredith** [comment]
Taking review :facepunch:

**Comment by hxtomprice** [comment]
Reviewing

**Comment by connormeredith** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -7,7 +7,41 @@
 
 # jsonapi-server
 
-`jsonapi-server` is a fully featured NodeJS server implementation of `json:api`. You provide the resources, we provide the api.
+A config driven NodeJS framework implementing [`json:api`](http://jsonapi.org/). You define the resources, it provides the api.
+
+### Motivation / Justification / Rationale
+
+This framework solves the challenge of json:api without coupling us to any one ORM solution. Every other module out there is either tightly coupled to a database implementation, tracking an old version of the json:api spec, or is merely a helper library for a small feature. If you're building an API and your use case only involves reading and writing to a data store... well count yourself lucky. For everyone else, this framework provides the flexibility to provide complex a API without wasting developer time focusing on anything other than shipping valuable features.
```
Comment (hxtomprice) [inline README.md]: `to provide complex a API` should be `to provide a complex API

**Comment by hxtomprice** [comment]
:+1: :8ball:

**Pull request merged**